webview_panels/src/modules/documentationEditor/components/documentationPropagation/DocumentationPropagation.tsx

```
@@ -0,0 +1,270 @@
+      })) as { saved: boolean };
+      if (!result.saved) {
+        panelLogger.error("Unable to save documentation", item);
+      }
+    }
+  };
+
+  const setAllColumnsValue = (value: boolean) => {
+    setSelectedColumns(
+      allColumns.reduce(
+        (acc, curr) => ({
+          ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)